packages/governance/src/types.js

```diff
@@ -244,11 +244,26 @@
  * @typedef {Object} ElectoratePublic
  * @property {() => Subscription<QuestionDetails>} getQuestionSubscription
  * @property {() => Promise<Handle<'Question'>[]>} getOpenQuestions,
- * @property {() => string} getName
  * @property {() => Instance} getInstance
  * @property {(h: Handle<'Question'>) => Promise<Question>} getQuestion
  */
 
+/**
+ * @typedef {Object} CommitteeElectoratePublicMixin
+ * @property {() => string} getName
+ */
+
+/**
+ * @typedef {Object} ClaimsElectorateMixin
+ * @property {() => ERef<Invitation>} makeVoterInvitation
+ */
+
+/**
+ * @typedef { ElectoratePublic & ClaimsElectorateMixin } ClaimsElectoratePublic
+ * @typedef { ElectoratePublic & CommitteeElectoratePublicMixin } CommitteeElectoratePublic
+ * @typedef { ClaimsElectoratePublic | CommitteeElectoratePublic } AnyElectoratePublic
+ */
```

Comment:
Making up all these names for things we only use once only makes the code harder to read, to my mind. I suggest:

```suggestion
/**
 * @typedef { ElectoratePublic & {makeVoterInvitation: () => ERef<Invitation>} } ClaimsElectoratePublic
 * @typedef { ElectoratePublic & {getName: () => string} } CommitteeElectoratePublic
 */
```

Reply:
That is simpler for this case. It feels like if there were more than one added method, we'd need to break it out for readability anyway.  Do you think the in-line approach would apply more generally?

Reply:
> Do you think the in-line approach would apply more generally?

I do, though I gather it goes against an established norm to some extent.